x-pack/solutions/security/test/cloud_security_posture_api/routes/graph.ts

```diff
@@ -694,6 +697,247 @@ export default function (providerContext: FtrProviderContext) {
+        const enrichPolicyCreationTimeout = 10000;
+
+        before(async () => {
+          await kibanaServer.uiSettings.update({ 'securitySolution:enableAssetInventory': true });
+          await esArchiver.load(
+            'x-pack/solutions/security/test/cloud_security_posture_api/es_archives/entity_store'
+          );
+        });
+
+        after(async () => {
+          await esArchiver.unload(
+            'x-pack/solutions/security/test/cloud_security_posture_api/es_archives/entity_store'
+          );
+          await kibanaServer.uiSettings.update({ 'securitySolution:enableAssetInventory': false });
+        });
+
+        it('should return 200 when securitySolution:enableAssetInventory is true without enrich policy', async () => {
+          const response = await postGraph(supertest, {
+            query: {
+              originEventIds: [],
+              start: '2024-09-01T00:00:00Z',
+              end: '2024-09-02T00:00:00Z',
+              esQuery: {
+                bool: {
+                  filter: [
+                    {
+                      match_phrase: {
+                        'actor.entity.id': 'admin@example.com',
+                      },
+                    },
+                  ],
+                  must_not: [
+                    {
+                      match_phrase: {
+                        'event.action': 'google.iam.admin.v1.UpdateRole',
+                      },
+                    },
+                  ],
+                },
+              },
+            },
+          }).expect(result(200));
+
+          expect(response.body).to.have.property('nodes').length(3);
+          expect(response.body).to.have.property('edges').length(2);
+        });
+
+        it('should contain entity data when asset inventory is enabled', async () => {
+          // initialize security-solution-default data-view - needed for entity store
+          const dataView = dataViewRouteHelpersFactory(supertest);
+          await dataView.create('security-solution');
+
+          // enable asset inventory - install underlying indexes and enrich policies
+          await supertest
+            .post(`/api/asset_inventory/enable`)
+            .set('kbn-xsrf', 'xxxx')
+            .send({})
+            .expect(200);
+
+          const data = await supertest
+            .get(`/api/fleet/epm/packages/cloud_asset_inventory`)
+            .set('kbn-xsrf', 'xxxx')
+            .send({})
+            .expect(200);
+
+          // although enrich policy is already create via 'api/asset_inventory/enable'
+          // we still would like to replicate as if cloud asset discovery integration was fully installed
+          await supertest
+            .post(`/api/fleet/epm/packages/_bulk`)
+            .set('kbn-xsrf', 'xxxx')
+            .send({
+              packages: [
+                {
+                  name: 'cloud_asset_inventory',
+                  version: data.body.item.version,
```

Comment:
Pay attention you install the latest version - the test will be skipped as soon as there will be a bug in the integration.
The way we deal with it today is to have a fixed version we test with, which we update manually